Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=no-member
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+try:
+    import demisto_ml
+except Exception:
+    pass
+
+
+def get_model_data(model_name, store_type):
+    if store_type == 'list':
+        res = demisto.executeCommand("getList", {"listName": model_name})
+        res = res[0]
+        if is_error(res):
+            return_error("error reading list %s from Demisto" % model_name)
+        return res["Contents"]
+    elif store_type == 'mlModel':
+        res = demisto.executeCommand("getMLModel", {"modelName": model_name})
+        res = res[0]
+        if is_error(res):
+            return_error("error reading model %s from Demisto" % model_name)
+        return res['Contents']['modelData']
+
+
+def predict_phishing_words(model_name, model_store_type, email_subject, email_body):
+    model_data = get_model_data(model_name, model_store_type)
+    model = demisto_ml.decode_model(model_data)
+    text = "%s %s" % (email_subject, email_body)
+    res = demisto.executeCommand('WordTokenizerNLP', {'value': text,
+                                                      'hashWordWithSeed': demisto.args().get('hashSeed')})
+    if is_error(res[0]):
+        return_error(res[0]['Contents'])
+    tokenized_text_result = res[0]['Contents']
+    input_text = tokenized_text_result['hashedTokenizedText'] if tokenized_text_result.get('hashedTokenizedText') else \
+        tokenized_text_result['tokenizedText']
+    filtered_text, filtered_text_number_of_words = demisto_ml.filter_model_words(input_text, model)
+    if filtered_text_number_of_words == 0:
+        return_error("The model does not contains any of the input text words")
```

Comment:
why is that an error?
`contains` => `contain`

Reply:
I check the input text after pre-process, and lemmatization will be applied, so "contains" will convert to contain after the pre-process